en/services/component_information.md

```diff
@@ -0,0 +1,91 @@
+# Component Information Protocol
+
+> **Note** This service is still marked as "work in progress", and should not be relied upon in production setups.
+
+The *Component Information Protocol* is a MAVLink service for requesting information from (and about) MAVLink components. 
+It is intended to provide autopilot- and version- independent feature discovery and configuration, allowing a GCS to configure its UI and/or a device without knowing anything about the connected system.
+
+Information shared using this service may include:
+- What types of component information are supported (by this component).
+- What MAVLink commands are supported (both in missions and in other modes).
+- Parameter metadata for parameters supported by the vehicle.
+- Metadata about events emitted by the system
+- Self-describing configuration UIs (i.e. similar to MAVLink camera configuration files).
+- Translations of other metadata.
+
+The service interface uses the [COMPONENT_INFORMATION](#COMPONENT_INFORMATION) message to deliver information about the location of JSON-format [component information files](#schema_files) (and associated translation files).
+
+Component information is requested using [MAV_CMD_REQUEST_MESSAGE](#MAV_CMD_REQUEST_MESSAGE), specifying the [COMPONENT_INFORMATION](#COMPONENT_INFORMATION) message and [type](#COMP_METADATA_TYPE) of component information required.
+
+> **Note**  The `COMP_METADATA_TYPE_VERSION` type provides information about the data types the component supports, and should therefore be requested first.
+
+The `COMPONENT_INFORMATION` message provides a URL to where a file containing the requested information can be downloaded, along with a uid that can be used to determine if the file has changed since last downloaded (effectively a hash for the file). 
+The URL is either a HTTP or MAVLink FTP address; the second is preferred because the file supplied by a vehicle can always be assumed to be up to date.
+
+In addition, the `COMPONENT_INFORMATION` may include a URL and uid for a translation file.
+If supplied, this contains translation strings for the information file specified in the message, in JSON translation file format (superseding any translations in the information file).
+
+Information supplied by the service is assumed to be invariant after boot.
+There is no mechanism, for example, to provide an update if the set of supported parameters was to change after boot.
+
+> **Note** Component information files may be **.gz** compressed.
```

Comment:
@bkueng Should it also say .lzma is supported even though it is legacy format?

Reply:
I'd rather explicitly say it's not supported, as the embedded library does not have support for that.

Reply:
Thanks busys. The docs only say what **is** supported (no one reading this will assume lmza might be supported for thsi). I have updated this in a different push.